Microsoft.Toolkit.Uwp.SampleApp/App.xaml

```diff
@@ -190,6 +190,7 @@
             <Setter Property="UseSystemFocusVisuals" Value="True" />
             <Setter Property="HorizontalContentAlignment" Value="Left" />
             <Setter Property="VerticalContentAlignment" Value="Stretch" />
+            <Setter Property="UseSystemFocusVisuals" Value="False" />
```

Comment:
This is a duplicate. Also, it should be set to true in order for the focus rects to show up on the hamburger menu.